common/lib/xmodule/xmodule/templates/problem/multiplechoice.yaml

```diff
@@ -2,51 +2,31 @@
 metadata:
     display_name: Multiple Choice
     markdown: |
-       Multiple choice problems allow learners to select only one option. Learners can see all the options along with the problem text.
 
-       When you add the problem, be sure to select Settings to specify a Display Name and other values that apply.
+        You can use this template as a guide to the simple editor markdown and OLX markup to use for multiple choice with hints and feedback problems. Edit this component to replace this template with your own assessment.
```

Comment:
this template is just for multiple choice (without hints and feedback) correct?
If that is correct, please remove "with hints and feedback"